perceval/serialization/deserialize.py

```diff
@@ -145,38 +158,22 @@ def deserialize(obj):
         r = []
         for k in obj:
             r.append(deserialize(k))
-    elif isinstance(obj, str) and obj.startswith(":PCVL:"):
-        if obj.startswith(":PCVL:zip:"):
+    elif isinstance(obj, str) and obj.startswith(PCVL_PREFIX):
+        if obj.startswith(ZIP_PREFIX):
             # if zip found -> update obj
             # STEPS: remove prefix -> decode b64 encoding -> decompress -> decode utf-8 (byte-> str)
-            zip_prefix = ":PCVL:zip:"
-            obj = obj[len(zip_prefix):]
-            obj = b64decode(obj)
-            obj = (decompress(obj)).decode('utf-8')
-
-        p = obj[6:].find(":")
-        cl = obj[6:p+6]
-        sobj = obj[p+7:]
-        if cl == "BasicState":
-            r = BasicState(sobj)
-        elif cl == "StateVector":
-            r = deserialize_statevector(sobj)
-        elif cl == "SVDistribution":
-            r = deserialize_svdistribution(sobj)
-        elif cl == "BSDistribution":
-            r = deserialize_bsdistribution(sobj)
-        elif cl == "BSCount":
-            r = deserialize_bscount(sobj)
-        elif cl == "BSSamples":
-            r = deserialize_bssamples(sobj)
-        elif cl == "Matrix":
-            r = deserialize_matrix(obj)
-        elif cl == "ACircuit":
-            r = deserialize_circuit(obj)
-        elif cl == "NoiseModel":
-            r = deserialize_noise_model(sobj)
-        else:
-            raise NotImplementedError(f"No deserializer found for {cl}")
+            obj = b64decode(obj[len(ZIP_PREFIX):])
+            obj = decompress(obj).decode('utf-8')
+
+        p = obj[len(PCVL_PREFIX):].find(SEP)
+        class_obj = obj[6:p+6]
```

Comment:
I would suggest replacing the number '6' in the code, perhaps using `len(PCVL_PREFIX)` to allow for a quicker understanding.